src/test/java/uk/gov/hmcts/ccd/endpoint/ui/UserProfileEndpointTest.java

```diff
@@ -53,9 +53,9 @@ public void validUser() throws Exception {
         assertNull(userProfile.getChannels());
 
         final JurisdictionDisplayProperties[] jurisdictions = userProfile.getJurisdictions();
-        assertEquals(3, jurisdictions.length);
+        assertEquals(4, jurisdictions.length);
```

Comment:
Let's add an assertion for the first element too then

Reply:
added the assertion for the first element